static/src/javascripts/projects/common/modules/experiments/__fixtures__/ab-test.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type { ABTest, Variant } from '../ab-types';
+
+export const genVariant = (id: string): Variant => ({
+    id,
+    test: () => undefined,
+});
+
+export const genAbTest = (id: string, canRun: ?boolean): ABTest => ({
```

Comment:
Isn't this what [these](https://github.com/guardian/frontend/blob/4a2b32b9b8b862c677d407cc7699ae5ddef095e6/static/src/javascripts/projects/common/modules/experiments/ab.spec.js#L66) [tests](https://github.com/guardian/frontend/blob/4a2b32b9b8b862c677d407cc7699ae5ddef095e6/static/src/javascripts/projects/common/modules/experiments/ab.spec.js#L85) are doing?